core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -263,41 +261,6 @@ public BigQuerySqlDialect(SqlDialect.Context context) {
     }
   }
 
-  /**
-   * For usage of TRIM, LTRIM and RTRIM in BQ see
-   * <a href="https://cloud.google.com/bigquery/docs/reference/standard-sql/functions-and-operators#trim">
-   * BQ Trim Function</a>.
-   */
-  private static void unparseTrim(SqlWriter writer, SqlCall call, int leftPrec,
```

Comment:
I noticed that the logic of the `unparseTrim` has also been adjusted here. Do we need to wait for #4360 to complete before rebasing?

Reply:
Yes，here need wait to rebase~